### PR TITLE
Add a basic scripts directory for shipping job runner related commands

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: jobrunner/bin/update-jobrunner.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+ref=${1:-HEAD}
+
+cd /srv/jobrunner/code
+git fetch
```
Comment (madwort) [inline jobrunner/bin/update-jobrunner.sh:7]: this should be fine, but maybe `git fetch --all` is better?

File: jobrunner/bin/update-jobrunner.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+git checkout "$ref"
```
Comment (madwort) [inline jobrunner/bin/update-jobrunner.sh:8]: this should be fine, assuming the production repository remains clean - other options include cribbing [what openprescribing does](https://github.com/ebmdatalab/openprescribing/blob/main/fabfile.py#L76) (`git checkout --force "origin/$ref"`)

**Review by madwort — APPROVED** [review]
looks good, nitpick suggestions for possible enhancement

File: jobrunner/bin/update-jobrunner.sh (hunk 2 above)
Comment (bloodearnest) [reply jobrunner/bin/update-jobrunner.sh:8]: nice, will do

**Review by madwort — APPROVED** [review]

**Pull request merged**